packages/alea-frontend/pages/api/get-fau-clip-info/[clipId].ts

```diff
@@ -19,21 +19,31 @@ function isFresh(cachedObject: CachedObject) {
 export async function getVideoInfo(clipId: string): Promise<ClipDetails> {
   const url = `${FAU_TV_OEMBED_BASE_URL}?url=${FAU_TV_BASE_URL}/clip/id/${clipId}&format=json`;
   const { data } = await axios.get(url);
-//TODO: Hack: for now FAU TV api provides 360p,720p and 1080p url for only presenter view
-//TODO:  In both presenter+presentation view FAU TV provide only one url , so adding that only as of now.
+  //TODO: Hack: for now FAU TV api provides 360p,720p and 1080p url for only presenter view
+  //TODO:  In both presenter+presentation view FAU TV provide only one url , so adding that only as of now.
+  const presenterUrl: string | undefined =
+    (data && (data.presenter_url as string)) || (data && (data.file as string)) || undefined;
```

Comment:
This and other assignments in this bllock are unnecessarily verbose.
```suggestion
  const presenterUrl: string | undefined = data?.presenter_url || data?.file;
```